kwords.py

```diff
@@ -0,0 +1,34 @@
+import requests
+import json
+
+url = "http://0.0.0.0:5000/"
+
+def get_keywords(text):
+
+    payload = json.dumps({
+    "text": text,
+    "lang": "en",
+    "n": 10
+    })
+    headers = {
+    'Content-Type': 'application/json'
+    }
+
+
+    # obtain response
+    response = requests.request("POST", url, headers=headers, data=payload)
+
+    keywords, scores, relevant_keywords = response.json()
+
+    # Pair each keyword with its corresponding score
+    keyword_score_pairs = list(zip(keywords, scores))
+
+    # Sort the pairs based on the scores (in descending order)
+    sorted_keyword_score_pairs = sorted(keyword_score_pairs, key=lambda x: x[1], reverse=True)
+
+    # Extract the sorted keywords and scores separately
+    sorted_keywords, sorted_scores = zip(*sorted_keyword_score_pairs)
+
+    return {"keywords":list(sorted_keywords[:8])}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `get_keywords` does not handle potential exceptions that could occur during the execution of the requests.request method (like a TimeoutError, ConnectionError, etc.). It's important to add error handling to ensure the function doesn't crash the entire program when it encounters an issue with the request.
```diff
  def get_keywords(text):
      ...
      try:
          response = requests.request("POST", url, headers=headers, data=payload)
          response.raise_for_status()
      except requests.exceptions.RequestException as e:
          print(f"An error occurred: {e}")
          return None
      ...
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->